test/image/test_load_image.py

```diff
@@ -42,7 +42,7 @@ def test_load_invalid_base64_data():
 
 
 # ensure memory usage does not get too high - this value depends on Python version:
-@memunit.assert_lt_mb(141)
+@memunit.assert_lt_mb(200)
```

Comment:
Please do not bump this above 150MB, it's not necessary

Reply:
What's the point?
The demand will rise with every new test added, and that seems unlikely to get fixed in pytest any time soon.
Is there any real disadvantage of adding some extra headroom here?

Reply:
I'm still not sure this is due to `pytest`... Investigation is ongoing in #641

The point is that keeping those thresholds allows me to keep track of how fast / what causes memory usage increases.
So far they helped me to link the growing number of tests with this memory usage, but this could be [spurious-correlation](https://www.tylervigen.com/spurious-correlations)...
As long as the increasing memory usage problem is not solved, I'd prefer to keep those checks a low as possible, in hope this will help pinpoint the origin of the issue